setup.cfg

```diff
@@ -57,7 +57,7 @@ install_requires =
     tqdm>=4.38.0,<5.0.0
     numpy>=1.15.0
     requests>=2.13.0,<3.0.0
-    pydantic>=1.7.4,!=1.8,!=1.8.1,<1.9.0
+    pydantic>=1.7.4,!=1.8,!=1.8.1,<1.10.0
```

Comment:
@adrianeboyd: do you remember why we have these specified both in `thinc` as well as in `spacy`?

Reply:
Both `thinc` and `spacy` include `pydantic` imports.

Reply:
Yes, but why? Why doesn't `spacy` just rely on `Thinc` pulling in `pydantic`?

Reply:
I can see an argument in favour of this technique to have slightly more independent release cycles between the two repo's, but in general it does feel like we're just creating overhead / duplicate work for ourselves :|